README.md

```diff
@@ -2,8 +2,38 @@
 
 This is the source code for the Scala.js website at
 [http://www.scala-js.org/](http://www.scala-js.org/).
-It is built with [Jekyll](http://jekyllrb.com/).
-Pull requests welcome!
 
-Looking for the source code of Scala.js itself?
-It is [over there](https://github.com/scala-js/scala-js).
+If you intended to look at the Scala.js source code itself,
+you can find it here: [github.com/scala-js/scala-js](https://github.com/scala-js/scala-js).
+
+# Contributing
+
+The key to contributing is being able to edit and
+preview your content. [Your pull requests are welcome](https://github.com/scala-js/scala-js-website/compare)!
+
+## Set up
+As this website is built with [Jekyll](http://jekyllrb.com/),
+we will need to set up some Ruby tooling.
+
+First, install RVM (Ruby Version Manager): https://rvm.io/rvm/install
+Then run the following commands:
+```bash
+$ source ~/.rvm/scripts/rvm
+# Use Ruby 2.2.3
+$ rvm use 2.2.3 --install --fuzzy
+# Set up Bundler, a Ruby package manager
+# It downloads dependencies specified in a Gemfile
+# but into a local path unlike gem 
+$ gem install bundler
+# Install dependencies such as Jekyll and its plugins
+$ bundle install
```

Comment:
This step didn't work for me:
```
doeraene@lamppc18:~/projects/scalajs-website$ bundle install
Fetching gem metadata from https://rubygems.org/...........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Using addressable 2.3.8
Using bundler 1.16.5
Using colorator 0.1
Using concurrent-ruby 1.0.5
Using execjs 2.6.0
Using fastimage 1.9.0
Fetching ffi 1.9.25
Installing ffi 1.9.25 with native extensions
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

/home/doeraene/.rvm/rubies/ruby-2.2.3/bin/ruby -r ./siteconf20181002-17905-9x53re.rb
extconf.rb
checking for ffi.h... *** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
        --with-opt-dir
        --without-opt-dir
        --with-opt-include
        --without-opt-include=${opt-dir}/include
        --with-opt-lib
        --without-opt-lib=${opt-dir}/lib
        --with-make-prog
        --without-make-prog
        --srcdir=.
        --curdir
        --ruby=/home/doeraene/.rvm/rubies/ruby-2.2.3/bin/$(RUBY_BASE_NAME)
        --with-ffi_c-dir
        --without-ffi_c-dir
        --with-ffi_c-include
        --without-ffi_c-include=${ffi_c-dir}/include
        --with-ffi_c-lib
        --without-ffi_c-lib=${ffi_c-dir}/lib
        --with-libffi-config
        --without-libffi-config
        --with-pkg-config
        --without-pkg-config
/home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:456:in `try_do': The compiler
failed to generate an executable file. (RuntimeError)
You have to install development tools first.
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:587:in `try_cpp'
from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:1060:in `block in
have_header'
from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:911:in `block in
checking_for'
from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:351:in `block (2 levels)
in postpone'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:321:in `open'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:351:in `block in postpone'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:321:in `open'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:347:in `postpone'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:910:in `checking_for'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:1059:in `have_header'
        from extconf.rb:16:in `<main>'

extconf failed, exit code 1

Gem files will remain installed in /home/doeraene/.rvm/gems/ruby-2.2.3/gems/ffi-1.9.25 for
inspection.
Results logged to
/home/doeraene/.rvm/gems/ruby-2.2.3/extensions/x86_64-linux/2.2.0/ffi-1.9.25/gem_make.out

An error occurred while installing ffi (1.9.25), and Bundler cannot continue.
Make sure that `gem install ffi -v '1.9.25' --source 'https://rubygems.org/'` succeeds before
bundling.

In Gemfile:
  jekyll-assets was resolved to 2.1.2, which depends on
    jekyll was resolved to 3.0.1, which depends on
      jekyll-watch was resolved to 1.5.1, which depends on
        listen was resolved to 3.1.5, which depends on
          rb-inotify was resolved to 0.9.10, which depends on
            ffi
doeraene@lamppc18:~/projects/scalajs-website$ gem install ffi -v '1.9.25' --source 'https://rubygems.org/'
Building native extensions.  This could take a while...
ERROR:  Error installing ffi:
        ERROR: Failed to build gem native extension.

    /home/doeraene/.rvm/rubies/ruby-2.2.3/bin/ruby -r ./siteconf20181002-18022-1ev8wcs.rb extconf.rb
checking for ffi.h... *** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
        --with-opt-dir
        --without-opt-dir
        --with-opt-include
        --without-opt-include=${opt-dir}/include
        --with-opt-lib
        --without-opt-lib=${opt-dir}/lib
        --with-make-prog
        --without-make-prog
        --srcdir=.
        --curdir
        --ruby=/home/doeraene/.rvm/rubies/ruby-2.2.3/bin/$(RUBY_BASE_NAME)
        --with-ffi_c-dir
        --without-ffi_c-dir
        --with-ffi_c-include
        --without-ffi_c-include=${ffi_c-dir}/include
        --with-ffi_c-lib
        --without-ffi_c-lib=${ffi_c-dir}/lib
        --with-libffi-config
        --without-libffi-config
        --with-pkg-config
        --without-pkg-config
/home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:456:in `try_do': The compiler failed to generate an executable file. (RuntimeError)
You have to install development tools first.
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:587:in `try_cpp'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:1060:in `block in have_header'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:911:in `block in checking_for'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:351:in `block (2 levels) in postpone'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:321:in `open'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:351:in `block in postpone'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:321:in `open'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:347:in `postpone'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:910:in `checking_for'
        from /home/doeraene/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/mkmf.rb:1059:in `have_header'
        from extconf.rb:16:in `<main>'

extconf failed, exit code 1

Gem files will remain installed in /home/doeraene/.rvm/gems/ruby-2.2.3/gems/ffi-1.9.25 for inspection.
Results logged to /home/doeraene/.rvm/gems/ruby-2.2.3/extensions/x86_64-linux/2.2.0/ffi-1.9.25/gem_make.out
```

Reply:
https://github.com/ffi/ffi/issues/608#issuecomment-363698071